task2/build/webpack.base.config.js

```diff
@@ -0,0 +1,44 @@
+const path = require('path');
+const webpack = require('webpack');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+const parentDir = path.join(__dirname, '../');
```

Comment:
When you have webpack config in the folder other than the root one, it is good to use process.cwd() as an entry instead.

Reply:
Make all sense. done :)